Makefile

```diff
@@ -100,12 +100,14 @@ imports: $(OPENSHIFT_GOIMPORTS)
 COUNT ?= 5
 
 .PHONY: test-e2e
+test-e2e: WHAT ?= ./test/e2e...
 test-e2e: install
-	go test -tags e2e -race -count $(COUNT) ./test/e2e... $(WHAT)
```

Comment:
Someone will be very surprised when they choose something other than e2e here and the build tags mean their `make` target does nothing :)

Reply:
what do you suggest? Without this WHAT does not make sense either.

Reply:
where exactly do we use the tag today?